### DISCUSSION TITLE
A single space in a file name hides all file names?

### DISCUSSION THREAD

**Discussion by Marxsal** [gsantner/markor#2135] [Q&A] [answered]
I don't know if this is a known behavior or not, and the bug report requires that I load a development version, which is just a little too much.

If I have a single file with a space in it's name, then all the files that appear with "Files" are missing file names.  They show dates, but no file name. I could understand if the one file with spaces appeared as blank, but to have all of them disappear seems incorrect.

Thanks!

**Comment by gsantner** [comment, marked as answer]
> the bug report requires that I load a development version, which is just a little too much.

Why is it too much? Please take your time to check if it still applies in the latest development version (=master).    

If you don't want to take the time to verify the issue and follow the project reporting scheme ... please don't expect somebody to take his time either to look at your text.

Thank you.
